src/plugins/links/public/components/links_component.tsx

```diff
@@ -62,6 +82,10 @@ export const LinksComponent = () => {
       }`}
       paddingSize="xs"
       data-test-subj="links--component"
+      data-shared-item
+      data-render-complete
```

Comment:
Do we still need these? :eyes: Shouldn't the render complete dispatched take care of this?

Reply:
Oops, yeah. This shouldn't be there anymore. I should have reverted my earlier changes before picking this up again. Fixed in https://github.com/elastic/kibana/pull/167896/commits/bb8a0396694b09700821ace0fe02bc08d5ec3b00